phpunit/block-editor-test.php

```diff
@@ -243,7 +244,7 @@ function test_get_default_block_editor_settings() {
 	 */
 	function test_get_block_editor_settings_returns_default_settings() {
 		$this->assertSameSets(
-			gutenberg_get_block_editor_settings( 'post-editor' ),
+			gutenberg_get_block_editor_settings( 'my-editor' ),
```

Comment:
When using `post-editor` all hooks from the Gutenberg plugin get applied as well.

Reply:
Why this change is need though?

Reply:
At the moment the wrapper function returns the same structure as the default block editor settings. `post-editor` has special handling so it applies all existing filters present in the Gutenberg plugin.

Reply:
So it's just a change to not apply filters that we don't need for these tests or the filters interfere with the output? I'm not familiar with this :)

Reply:
Yes, those filters introduce noise for tests so I found a way to ignore them 😄 Eventually those tests will be run only in WP core and it won't be an issue anymore.

Reply:
Thanks for explaining Grzegorz!